features/payment/managing_payment_methods/editing_payment_method.feature

```diff
@@ -0,0 +1,48 @@
+@managing_payment_methods
+Feature: Editing payment methods
+    In order to change which payment methods are available in my store
+    As an Administrator
+    I want to be able to edit payment method
+
+    Background:
+        Given the store has a payment method "Offline" with a code "offline"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Renaming the payment method
+        Given I want to modify a payment method "Offline"
+        When I rename it to "Cash on delivery" in "English (United States)"
+        And I save my changes
+        Then I should be notified about successful edition
+        And this payment method name should be "Cash on delivery"
+
+    @ui
+    Scenario: Changing gateway
```

Comment:
Extract it to two different scenarios. `Modifying payment method gateway` and `Renaming payment method`